data/store.ts

```diff
@@ -70,12 +77,20 @@ const pricesPersistConfig = {
   whitelist: ["currencySelected"]
 };
 
+const settingsPersistConfig = {
```

Comment:
Persist config not required, instead add `settings` to the higher-level persistConfig on line `64`